external/js/cothority/lib/net/net.js

```diff
@@ -3,27 +3,28 @@
 const topl = require('topl');
 const UUID = require('pure-uuid');
 const protobuf = require('protobufjs')
+const co = require('co');
+const shuffle = require("crypto-shuffle");
```

Comment:
single quotes (does this matter to JS, or is it just for OCD people like me?)

Reply:
Since I installed the eslint-er it seems everything is put as double quotes. And from what I know it does not matter.